lib/liftoff/cli.rb

```diff
@@ -7,7 +7,7 @@ def initialize(argv)
 
     def run
       parse_command_line_options
-      LaunchPad.new.liftoff @options
+      LaunchPad.new.liftoff @options, @argv.first
```

Comment:
Sure, but most people won't. The path passed on the command line will take precedence anyway, so I don't think it's that big of a deal. I'm more interested in simplifying this code path than preventing a weird customization.